src/main/java/com/revature/g2g/models/SurveyDTO.java

```diff
@@ -2,12 +2,24 @@
 
 import java.util.List;
 
+import com.revature.g2g.api.templates.SurveySkillTemplate;
+
 import lombok.Data;
 import lombok.NoArgsConstructor;
 
 @Data @NoArgsConstructor
 public class SurveyDTO {
-	private List<Player> players;
-	private List<Skill> skills;
-	private Room room;
+	private List<PlayerDTO> players;
+	private List<SkillDTO> skills;
+	private RoomDTO room;
+
+	public SurveyDTO(PlayerDTO player, SurveySkillTemplate[] array) {
+		super();
+		this.players.add(player);
+		// TODO: SurveySkillTemplate is depreciated, will become SurveySkillDTO
+		for (SurveySkillTemplate sst: array) {
+			this.skills.add(new SkillDTO(sst.getSkill()) );
+		}
+		// TODO: figure out how room is assigned
+	}
```

Comment:
What is the purpose of this constructor? This might be a deprecated class that can be removed. We should check which DTO the SurveyController uses (after removing the code that the git merge accidentally had you put back in) and get rid of the rest.